src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+/*
+ * Tagging object class, to record list of commands for tags used in session
+ */
+
+public class Tagging {
+
+	public static final String _ADD = "+"; 
+	public static final String _REMOVE = "-";
```

Comment:
Naming is not consistent with other constants in the code base.
Should they be public?